index.html

```diff
@@ -355,6 +356,158 @@ <h3>Manifest Contexts</h3>
 						<code>https</code> when referencing Schema.org in the manifest.</p>
 			</section>
 
+			<section id="manifest-lang-dir">
+				<h3>Manifest Language and Direction</h3>
+
+				<p>Each each natural language property value in a manifest (e.g., <a href="#pub-title">title</a>, <a
+						href="#creators">creators</a>) has a default natural <dfn>language</dfn>, which is the language
+					that it is expressed in (e.g., English, French, Chinese). It also has a natural <dfn>base
+						direction</dfn> in which it is written — the display direction, either left-to-right or
+					right-to-left.</p>
+
+				<p>The digital publication manifest includes entries to set both these concepts to aid user agents in
+					interpreting and presenting the metadata.</p>
+
+				<section id="manifest-lang-dir-global">
+					<h4>Global Declarations</h4>
+
+					<p>The default language is set by including a language declaration in the context using the <a
+							href="https://w3c.github.io/json-ld-syntax/#syntax-tokens-and-keywords"
+								><code>language</code> key</a> [[!json-ld]]:</p>
+
+					<aside class="example" title="Declaring French as the default language for the manifest">
+						<pre><code>{
+    "@context" : [
+        "https://schema.org",
+        "https://www.w3.org/ns/pub-context", 
+        {"language":"fr"}
+    ],
+    "name" : "Le roi René"
+}</code></pre>
+					</aside>
+
+					<p class="note">It is not necessary to specify the at-sign (<code>@</code>) in the key name when
+						declaring the language as the publication context maps the terms to each other.</p>
+
```

Comment:
I am not sure this is something we should emphasize. While this is true, we do not do a similar remark v.a.v., e.g., `type` vs. `@type`. Maybe it is worth to emphasize, on the other hand, that order of these is important, ie, that the `language` should follow the pub context (which, technically, does these mappings).